### ISSUE TITLE
Tutorial doesn't work on GPU

### ISSUE THREAD

**Issue by cristipp** [allenai/allennlp#2555]
**Describe the bug**
The tutorial code at https://allennlp.org/tutorials doesn't work out of the box on GPU. For reference, to make it work:

```
    if torch.cuda.is_available():
        cuda_device = 0
        model = model.cuda(cuda_device)
    else:
        cuda_device = -1

    trainer = Trainer(model=model,
                      optimizer=optimizer,
                      iterator=iterator,
                      train_dataset=train_dataset,
                      validation_dataset=validation_dataset,
                      patience=10,
                      num_epochs=1000,
                      cuda_device=cuda_device)
```

**To Reproduce**
1. Copy/Paste the tutorial code from https://allennlp.org/tutorials.
2. Add cuda_device=0 as an argument on Trainer ctor.
3. Run on a GPU-enabled machine.
4. Observe `RuntimeError: Expected object of backend CPU but got backend CUDA for argument #3 'index'`

**Expected behavior**
The model should train.

**System (please complete the following information):**
 - OS: Linux
 - Python version: 3.6.8
 - AllenNLP version: 0.8.2
 - PyTorch version: 1.0.1.post2

**Comment by remotejob** [comment]
Completely the same for me. As well I tried  downgrade torch=1.0.0 don't help

**Comment by remotejob** [comment]
Downgrade AllenNLP==0.8.0 help!!
So we must wait till 0.8.3 I hope :)

**Comment by schmmd** [comment]
Thanks @cristipp, hope you're doing well!  Closing since this was fixed by @brendan-ai2 in #2570.
